app/decorators/hash_diff_decorator/base.rb

```diff
@@ -19,13 +19,21 @@ def to_html
     def diff_val(val)
       if val.nil?
         "[nil]"
+      elsif val.is_a?(Numeric)
+        diff_numeric(val)
       elsif val.empty?
         "[empty]"
-      elsif val.is_a?(Numeric)
-        format('%.2f', val)
       else
         val.inspect
       end
     end
+
+    def diff_numeric(val)
+      if val.is_a?(Fixnum)
+        val.to_s
+      elsif val.is_a?(Numeric)
```

Comment:
I wasn't sure about the original motivation for using `Numeric` so I preserved that as the default, and just checked explicitly for the (more common?) case of `Fixnum` which includes all `id` values.

I'm inclined to leave the current behavior alone until we get a specific example of where it does not work (as was the case here).